vendor.conf

```diff
@@ -71,6 +71,7 @@ github.com/xeipuuv/gojsonschema 1d523034197ff1f222f6429836dd36a2457a1874
 golang.org/x/crypto 69ecbb4d6d5dab05e49161c6e77ea40a030884e1
 golang.org/x/oauth2 a6bd8cefa1811bd24b86f8902872e4e8225f74c4
 golang.org/x/time f51c12702a4d776e4c1fa9b0fabab841babae631
+google.golang.org/appengine 54a98f90d1c46b7731eb8fb305d2a321c30ef610 # v1.5.0
```

Comment:
ok, so it looks to build fine without this dependency

probably because the file has a `// +build appengine` build-tag (which is not set in our case)

@estesp @Random-Liu wdyt? should we keep it, or remove it?

Reply:
Would `vndr` bring this back in the future if we remove it?

If yes, I think it might be fine to leave it here. :)